README.md

```diff
@@ -80,6 +80,12 @@ Example usage:
 
 # print list of formated rules for use with `pending` in config file
 ./node_modules/.bin/ember-template-lint app/templates/application.hbs --print-pending
+
+# specify custom ignore pattern `['**/dist/**', '**/tmp/**', '**/node_modules/**']` by default
+./node_modules/.bin/ember-template-lint /tmp/template.hbs --ignore-pattern "**/foo/**" "**/bar/**"
```

Comment:
This confuses me a tad, I think the `**/bar/**` here is going to be the files to lint (not part of the ignore patterns), right?

Reply:
Nope, because we specify type of `--ignore-pattern` flag as an array, it allows us to list values separated by space (https://github.com/yargs/yargs/blob/master/docs/api.md#arraykey); I'm guessing it parses until EOL or encountering next flag with `--`

Reply:
Gotcha, thank you for the docs link! Seems like we'd need to then use `--` to separate the actual paths to be linted, maybe tweak this example to show that?

Reply:
Actually, can we set `'greedy-arrays': false` to the `.parserConfiguration` in the `yargs` options? I think this will make it have the behavior that I expected...

See https://github.com/yargs/yargs-parser#greedy-arrays

Reply:
Updated and removed a test that uses consecutive args

Reply:
Awesome, thank you! I pushed https://github.com/ember-template-lint/ember-template-lint/pull/1200/commits/5415760ee1e13bee219172672a60c05206a50aac to fixup this README example now that we require `--ignore-pattern` in front of each.